hedera-node/hedera-app-spi/src/testFixtures/java/module-info.java

```diff
@@ -11,4 +11,5 @@
 
     // Temporarily needed until FakePreHandleContext can be removed
     requires com.hedera.node.app.service.token;
+    requires com.swirlds.config;
```

Comment:
Instead of adding it here we should define it as `requires transitive com.swirlds.config` in `com.hedera.node.app.spi`

Reply:
Done